datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVAnalyzerRule.scala

```diff
@@ -136,4 +139,29 @@ class MVAnalyzerRule(sparkSession: SparkSession) extends Rule[LogicalPlan] {
     }
     catalogs
   }
+
+  /**
+   * Check if any segments are set for main table for Query. If any segments are set, then
+   * skip mv datamap table for query
+   */
+  def isSetMainTableSegments(catalogs: Seq[Option[CatalogTable]]): Boolean = {
```

Comment:
Change the method name isSegmentSetForMainTable and return true if segment is set

Reply:
done